packages/esm-patient-forms-app/src/forms-workspace/form-list/forms-list.component.tsx

```diff
@@ -0,0 +1,210 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import styles from './forms-list.scss';
+import isEmpty from 'lodash-es/isEmpty';
+import { CardHeader, PatientChartPagination } from '@openmrs/esm-patient-common-lib';
+import { useConfig, usePagination, Visit, formatDatetime, isDesktop, useLayoutType } from '@openmrs/esm-framework';
+import { launchFormEntryOrHtmlForms } from '../../form-entry-interop';
+import {
+  Button,
+  Layer,
+  Tile,
+  DataTable,
+  DataTableHeader,
+  DataTableRow,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from '@carbon/react';
+import { Edit } from '@carbon/react/icons';
+import { CompletedFormInfo, FormsSection } from '../../types';
+import { ConfigObject, OrderBy } from '../../config-schema';
+import { defaultPaginationSize } from '../../constants';
+import { HtmlFormEntryForm } from '@openmrs/esm-patient-forms-app/src/config-schema';
+
+export interface FormsListProps {
+  patientUuid: string;
+  patient: fhir.Patient;
+  visit: Visit;
+  formsSection: FormsSection;
+  searchTerm: string;
+  pageSize: number;
+  htmlFormEntryForms: Array<HtmlFormEntryForm>;
+}
+
+const FormsList: React.FC<FormsListProps> = ({
```

Comment:
It would probably be a good idea to refactor things so that the `FormsList` was less directly tied to the workspace component. That way, we can use the same view in other places...

Reply:
What is our suggestion?
The way I see it only has what needs to work.

-   patientUuid: string;
-   patient: fhir.Patient;
-   visit: Visit;
-   formsSection: FormsSection;
-   searchTerm: string;
-   pageSize: number;
-   htmlFormEntryForms: Array<HtmlFormEntryForm>;

I could simplify the `formsSection`, but I would still need the information that is inside it to work.
Maybe add an option to not show the title?

Reply:
After a talk with @ibacher, we decided to ignore this.